src/core.js

```diff
@@ -22,6 +36,19 @@ jQuery.fn.init = function( arg1 ) {
 };
 jQuery.fn.init.prototype = jQuery.fn;
 
+jQuery.htmlPrefilter = function( html ) {
+	var changed;
+	if ( typeof html === "string" && html.trim().charAt( 0 ) === "<" ) {
```

Comment:
Not sure if this guard is needed but it shouldn't hurt.

Reply:
The original `jQuery.htmlPrefilter` in jQuery 3.4.1 & older is just:
```js
jQuery.htmlPrefilter = function( html ) {
	return html.replace( rxhtmlTag, "<$1></$2>" );
};
```
so it has no guard. Therefore, I don't think we need it here either.